assets/src/edit-story/components/panels/design/videoProcessing/videoProcessing.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { __ } from '@web-stories-wp/i18n';
+import styled from 'styled-components';
+import { useCallback } from 'react';
+import { useFeature } from 'flagged';
+
+/**
+ * Internal dependencies
+ */
+
+import { Row as DefaultRow } from '../../../form';
+import { SimplePanel } from '../../panel';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+} from '../../../../../design-system';
+import { useCommonObjectValue } from '../../shared';
+import { useLocalMedia } from '../../../../app/media';
+import { useConfig } from '../../../../app/config';
+
+const DEFAULT_RESOURCE = {
+  id: null,
+  src: null,
+  poster: null,
+  mimeType: null,
+  isOptimized: false,
+  alt: '',
+  title: '',
+};
+
+const Row = styled(DefaultRow)`
+  margin-top: 2px;
+`;
+
+function VideoProcessingPanel({ selectedElements }) {
```

Comment:
Is there a ticket to remove this again?

Reply:
CC @BrittanyIRL

Reply:
On it! This was just in my brain but now it is a ticket. 🤝 

https://github.com/google/web-stories-wp/issues/7393